From 4482f9207027de8f36630737ae085110896ea849 Mon Sep 17 00:00:00 2001
From: John Blackbourn <johnbillion@git.wordpress.org>
Date: Thu, 26 Jan 2017 13:41:44 +0000
Subject: [PATCH] Posts, Post Types: When using Excerpt mode on the Posts list
 table, ensure the excerpt output matches what was manually entered into the
 Excerpt field.

Built from https://develop.svn.wordpress.org/trunk@39956


git-svn-id: http://core.svn.wordpress.org/trunk@39893 1a063a9b-81f0-0310-95a4-ce76da25c4cd
---
 wp-admin/includes/class-wp-posts-list-table.php | 2 +-
 wp-includes/version.php                         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wp-admin/includes/class-wp-posts-list-table.php b/wp-admin/includes/class-wp-posts-list-table.php
index a0e4bfc8df6c..668c3d20a4c7 100644
--- a/wp-admin/includes/class-wp-posts-list-table.php
+++ b/wp-admin/includes/class-wp-posts-list-table.php
@@ -956,7 +956,7 @@ public function column_title( $post ) {
 		echo "</strong>\n";
 
 		if ( ! is_post_type_hierarchical( $this->screen->post_type ) && 'excerpt' === $mode && current_user_can( 'read_post', $post->ID ) ) {
-			the_excerpt();
+			echo esc_html( get_the_excerpt() );
 		}
 
 		get_inline_data( $post );
diff --git a/wp-includes/version.php b/wp-includes/version.php
index c26e7a0fee7f..56646674243a 100644
--- a/wp-includes/version.php
+++ b/wp-includes/version.php
@@ -4,7 +4,7 @@
  *
  * @global string $wp_version
  */
-$wp_version = '4.8-alpha-39954';
+$wp_version = '4.8-alpha-39956';
 
 /**
  * Holds the WordPress DB revision, increments when changes are made to the WordPress DB schema.
